chat-commands.js

```diff
@@ -936,6 +936,7 @@ exports.commands = {
 			return;
 		}
 		if (!this.can('ban', null, room)) return false;
```

Comment:
I feel like this line should check that.

I mean, it doesn't make sense - if you can't talk, then you shouldn't be able to ``ban`` in that room.

tldr; maybe ``this.can`` needs a refactor.